config/locales/api/analytics/en.yml

```diff
@@ -0,0 +1,49 @@
+      follow_up: "Follow-up"
+      patient:
+        one: "Patient"
+        other: "Patients"
+      registered_patients: "Registered patients"
+      follow_up_patients: "Follow-up patients"
+      hypertension_controlled: "Hypertension controlled"
+      hypertension_control_desc: "How many patients with hypertension BP <140/90 in the last 3 months?"
+      last_12_months: "Last 12 months"
+      achievements: "Achievements"
+      hypertension_patients: "Hypertension patients"
+      notes: "Notes"
+      footer_note_1: "Data is for all activity at %{facility_name}. If data appears out of date, tap the \"Sync pending\" link on the home screen."
+      thank_you: "Thank You"
+      thank_you_note: "Thank you for all of your hard work treating patients with hypertension. Your work saves lives from heart attacks and strokes."
+      definition: "Definition"
+      registered_patients_desc: "Patients who were registered into Simple by staff at this facility. The patient's \"Preferred facility\" could be any facility."
+      follow_up_patients_desc: "Patients with a BP or blood sugar measure recorded at this facility. The follow-up patients count is not affected by where the patient was registered or what the patient's \"Preferred Facility\" is."
+      numerator: "Numerator"
+      denominator: "Denominator"
+      hypertension_cohort_report: "Hypertension cohort report"
+      hypertension_cohort_report_desc: "Cohort reports show the outcome for newly registered patients with hypertension in their first few months of treatment."
+      patients_with_controlled_bp: "Patients visited with BP &lt;140/90"
+      patients_with_uncontrolled_bp: "Patients visited with BP &ge;140/90"
+      patients_with_no_bp_measure: "Patients with no BP measure"
+      registered_in: "registered in"
+      result_from_last_visit_in: "Result from last visit in"
+      no_patients: "No patients"
+      cohort_report_footnote_1: "Latest follow-up visit at any facility in quarter after registration."
+      cohort_report_footnote_2: "All HTN patients assigned to this facility, registered in the quarter."
+      cohort_report_footnote_3: "If a patient has multiple visits, their last BP is used to calculate BP control."
+      all_hypertension_and_diabetes: "All hypertension and diabetes"
+      hypertension_controlled_numerator: "Patients with at least one BP measure recorded in the last 3 months at <em>any facility</em>. At the most recent visit the patient had BP&nbsp;&lt;140 and &lt;90."
+      hypertension_controlled_denominator: "All patients diagnosed with hypertension who have this facility as their \"Preferred Facility.\" Patients registered in the most recent 3 months are excluded, so the facility has the opportunity to get their BP controlled."
+      hypertension: "Hypertension"
+      diabetes: "Diabetes"
+      all_time: "All-time"
+      genders:
+        all: "All"
+        female: "Female"
+        male: "Male"
+        transgender: "Transgender"
```

Comment:
Can we add these to the `api/en.yml` file please? We've chosen to consolidate into a single file before, and that way we don't have to change any Transifex configuration.